src/vcpkg/binarycaching.cpp

```diff
@@ -273,7 +273,7 @@ namespace
         {
         }
 
-        static Path make_archive_subpath(const std::string& abi) { return Path(abi.substr(0, 2)) / (abi + ".zip"); }
+        static Path make_archive_subpath(StringView abi) { return Path(abi.substr(0, 2)) / abi.to_string() + ".zip"; }
```

Comment:
This is in general worse: if we're making a `std::string` anyway we may as well reuse the caller's. Note that `substr(0,2)` fits in the SSO and thus won't allocate.